resources/eng.ini

```diff
@@ -214,7 +214,7 @@ list.name=list
 list.alias=l
 list.desc=Shows all plots a player has
 list.usage=/p list [player: string]
-list.found=Plot {%0} at ({%1};{%2}),
+list.found="Plot {%0} at ({%1};{%2}), "
```

Comment:
Altering other language files is out of the scope of this PR and should not be included.

Reply:
Okay, I'm going to do that in the other PR